schema/time_test.go

```diff
@@ -19,14 +32,92 @@ func TestTimeValidate(t *testing.T) {
 		assert.NoError(t, err)
 		if assert.IsType(t, v, now) {
 			assert.True(t, now.Equal(v.(time.Time)), f)
+
+// timeValidateTest can be used for positive validation tests only; negative
+// tests should be hand-written to check for the correct errors.
+type timeValidateTest struct {
+	validator  schema.Time
+	input      interface{}
+	expectTime time.Time
+}
```

Comment:
TODO: rewrite this. Inspiration: in https://github.com/searis/schema/blob/master/type_time_test.go